theano/sandbox/cuda/blas.py

```diff
@@ -417,13 +419,14 @@ def __hash__(self):
         return hash(type(self)) ^ hash(self.inplace)
 
     def __setstate__(self, dct):
-        inplace = dct.get('inplace', True)
-        if inplace:
-            self.destroy_map = {0: [0]}
-        self.inplace = inplace
+        self.__dict__.update(dct)
```

Comment:
I think we should keep the old code. Some old pickled file with this op would have only inplace in dct and would not have destroy_map. So add it back with a comment that we need to keep it to unpickled correctly old graph.